apollo-client.js

```diff
@@ -0,0 +1,6 @@
+import { ApolloClient, InMemoryCache } from '@apollo/client'
+
+export const client = new ApolloClient({
+  uri: process.env.GATSBY_GRAPHCMS_API_URL,
```

Comment:
Why use the prefix gatsby instead of leaving the existing name?

Reply:
To allow browser to access it
https://www.gatsbyjs.com/docs/how-to/local-development/environment-variables/#accessing-environment-variables-in-the-browser